core/src/main/java/org/infinispan/configuration/internal/PrivateGlobalConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package org.infinispan.configuration.internal;
+
+import org.infinispan.commons.configuration.attributes.AttributeDefinition;
+import org.infinispan.commons.configuration.attributes.AttributeSet;
+import org.infinispan.configuration.serializing.SerializedWith;
+
+/**
+ * An internal configuration.
+ * <p>
+ * This is an internal configuration to be used by Infinispan modules when some advanced or ergonomic configuration is
+ * needed.
+ *
+ * @author Pedro Ruivo
+ * @since 9.0
+ */
+@SerializedWith(PrivateGlobalConfigurationSerializer.class)
```

Comment:
probably it makes sense to exclude this package from javadocs using the @private doclet